Dockerfile

```diff
@@ -15,10 +15,17 @@ RUN apt-get update && \
     # Use a fixed apt-get repo to stop intermittent failures due to flaky httpredir connections,
     # as described by Lionel Chan at http://stackoverflow.com/a/37426929/5881346
 RUN sed -i "s/httpredir.debian.org/debian.uchicago.edu/" /etc/apt/sources.list && \
-    apt-get update && apt-get install -y build-essential unzip cmake && \
-    # Work to upgrade to Python 3.7 can be found on this branch: https://github.com/Kaggle/docker-python/blob/upgrade-py37/Dockerfile
-    conda install -y python=3.6.6 && \
+    apt-get update && \
+    # Needed by vowpalwabbit & lightGBM (GPU build).
```

Comment:
By the way apparently conda updated the vowpalwabbit version to the one you said our users currently have, 8.8.1 (https://anaconda.org/conda-forge/vowpalwabbit), in case that's easier. But it looks like this is already working so maybe it's easier to leave since lightgbm needs it too (not sure if the conda version of lightgbm supports gpu).

Reply:
The conda version of lightgbm doesn't support GPU, you have to build it from source to get GPU support.

Great, that `vowpalwabbit` just released the latest version from conda. I will take a look in a separate PR whether we can use one of the new pre-built vowpalwabbitwheel

Reply:
Given that we need the libboost libraries for lightgbm, probably better to simply leave it as-is for now to avoid having to install them with conda which installs them at a different location which create duplicates.